packages/nocms-plugin-json-resources/test/createJsonResourceProvider.spec.js

```diff
@@ -0,0 +1,107 @@
+import assert from 'assert';
+import { createJsonResourceId } from '../lib/createJsonResourceProvider.js';
+import { activate } from '../src/index.js';
+
+const expectedResources = [{
+	id: '/fake/path/1',
+	inFile: 'fake/path/1',
+	outFile: 'fake/path/1',
+	mimeType: 'application/json',
+	type: 'json',
+	data: {
+		_id: '5d775b424974756c3bc704a3',
+		index: 0,
+		guid: '211e66ee-ec86-489d-b1de-ede8435a0058',
+		isActive: false,
+		balance: '$2,581.53',
+		picture: 'http://placehold.it/32x32',
+		age: 40,
+		eyeColor: 'brown',
+		name: 'Gertrude Hogan',
+		gender: 'female',
+		company: 'DANCERITY',
+		email: 'gertrudehogan@dancerity.com',
+		phone: '+1 (905) 548-2162'
+	}
+},
+{
+	id: '/fake/path/2',
+	inFile: 'fake/path/2',
+	outFile: 'fake/path/2',
+	mimeType: 'application/json',
+	type: 'json',
+	data: { hello: 'world' }
+}];
+
+const fakeFiles = {
```

Comment:
Samee

Reply:
This needs to be globally available for the fakeFunctions